test/Bruno/Altinn.AccessManagement/Manual Test Collection/RightsInternal/GetOfferedRevoke/Kasper Revoke Offered from Rimstad.bru

```diff
@@ -0,0 +1,58 @@
+meta {
+  name: Kasper Revoke Offered from Rimstad
+  type: http
+  seq: 5
+}
+
+post {
+  url: {{baseUrl}}/accessmanagement/api/v1/internal/{{auth_partyId}}/rights/delegation/offered/revoke
+  body: json
+  auth: none
+}
+
+headers {
+  Authorization: {{apiKey}}
+}
+
+body:json {
+  
+    {
+      "to": [
+        {
+          "id": "urn:altinn:person:partyid",
```

Comment:
If this is to be a valid test case for the RightsInternal API Offered Revoke as used from Altinn 2, this should use
"to": "urn:altinn:userid"

Reply:
fixed test

Reply:
Didn't actually fix test! Reopening this one.

Reply:
Added old "UserAttribute" to resolver and GetRequestToDeleteResource, and the test is fixed now.